scripts/ci/lib.sh

```diff
@@ -551,15 +556,42 @@ image_prefetcher_start() {
     # pull secret
     NAMESPACE=$ns make -C operator stackrox-image-pull-secret
 
-    # apply daemonset and configmap
+    # apply configmap, daemonset etc
     retry 5 true kubectl apply --namespace=$ns -f "$manifest"
     rm -f "$image_list" "$manifest"
 }
 
 image_prefetcher_await() {
     local ns="prefetch-images"
     local name="$1"
+    local image_prefetcher_metrics="$2"
     kubectl rollout status daemonset "$name" -n "$ns" --timeout 15m
+    # All images fetched, now retrieve metrics.
+    local attempt=0
+    local service="service/${name}-metrics"
+    while [[ -z $(kubectl -n "${ns}" get "${service}" -o jsonpath="{.status.loadBalancer.ingress}" 2>/dev/null) ]]; do
+        if [ "$attempt" -lt "10" ]; then
+            echo "Waiting for ${service} to obtain endpoint ..."
```

Comment:
In other places we have `info` for logs and `echo` for errors
```suggestion
            info "Waiting for ${service} to obtain endpoint ..."
```